code/modules/clothing/suits/armor.dm

```diff
@@ -537,6 +537,28 @@
 	stiffness = LIGHT_STIFFNESS
 	style = STYLE_NONE
 
+/obj/item/clothing/suit/storage/vest/technomancer_old
+	name = "reinforced Technomancer armor"
+	desc = "Technomancer League's ballistic armor, less protective against industrial hazards but better in a fight."
+	icon_state = "armor_engineering_old"
+	item_state = "armor_engineering_old"
+	blood_overlay_type = "armor"
+	armor = list(
+		melee = 9,
+		bullet = 9,
+		energy = 9,
+		bomb = 40,
+		bio = 0,
+		rad = 0
```

Comment:
9/9/9 with light slowdown, light stiffness, and no style is unusable.
I've already said this on the discord but I assume saying this here and leaving a comment will attract Firefox's attention.

Reply:
This armor already has massive negatives over regular techno gear considering it makes them actively worse at their job (no rad, no void protection).

If it's going to be completely useless in standard techno activities, then at least make it viable for a fight. Bare minimum would be 11 across the board. 

Making it worse than literally generic security armor is insane. You may as well just not add it back.

Reply:
There's better or equivalent round start armor in the game already

Reply:
To add the rest of what I said in discord - 

If you make a piece of equipment specialized to do one task (in this case combat) and it is bad at said task, why would anyone ever use it? Answer- they wouldn't. 

The roundstart argument doesnt even hold up because as techno you're gonna be setting up the engine, not getting into fights. Due to this having no rad resist, you're not gonna use it for that either. 
Once the engine is done, you are likely to already be outclassed in armor by the majority of the ship, which is to be expected when this is literally worse than the cheapest, most generic armor in the game. 
Furthermore, technos have no fucking upgrades to this and probably won't for a while so you're just stuck with this as your departmental "combat option."

If it is implemented in this state, it will virtually never to be used. The few extra points of armor aren't worth losing your rad and space resistance when you'll still have far from competitive armor. 

Like Beads said, there's already substantially better round-start armor options and they haven't ruined the balance of the game yet despite having been there for years. Mining voidsuit starts with a 13-10-8 with no slowdown and no stiffness. Paramedic armor is 7-10-10 with high style and no slowdown. If having half decent roundstart armor didn't fuck up the "careful" balance of the game it sure won't fuck it up now, especially when given to the MOST gear-starved major faction in the game.